[BUG] java.nio.file.FileSystemNotFoundException: Provider "vfs" not installed

**Context:**
- Playwright Version: 1.19.0
- Operating System: Linux
- Browser: N/A
- Extra: Playwright JARs are bundled inside a WAR and deployed into WildFly/JBoss application server v11.0.0.Final

<!-- CLI to auto-capture this info -->
<!-- npx envinfo --preset playwright --markdown -->

**Code Snippet**

```java
import com.microsoft.playwright.Browser;
import com.microsoft.playwright.Page;
import com.microsoft.playwright.Playwright;
import com.microsoft.playwright.options.Margin;
import com.microsoft.playwright.options.Media;

try (Playwright playwright = Playwright.create();
    Browser browser = playwright.chromium().launch()) {
  final String tmpFile = "/tmp/file-" + Instant.now().getEpochSecond() + ".html";
  Files.write(Paths.get(tmpFile), html.getBytes());
  try (final Page page = browser.newPage()) {
    page.navigate("file://" + tmpFile);
    final Page.PdfOptions pdfOptions = configurePageOptions(header, footer);
    rendered = page.pdf(pdfOptions);
  }
}
```

**Describe the bug**

Playwright returns an exception `Caused by: java.lang.RuntimeException: Failed to create driver` which is caused by `java.nio.file.FileSystemNotFoundException: Provider "vfs" not installed` as following (redacted due to verbosity).

It happens at the line `try (Playwright playwright = Playwright.create();`.

```
io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [rt.jar:1.8.0_152]
	at java.lang.Thread.run(Thread.java:748) [rt.jar:1.8.0_152]
Caused by: java.lang.RuntimeException: Failed to create driver
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:54) [driver-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.PlaywrightImpl.create(PlaywrightImpl.java:40) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:96) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:100) [playwright-1.19.0.jar:1.19.0]
	at au.com.infomedix.pdfkit.PlaywrightHtmlToPdf.render(PlaywrightHtmlToPdf.java:31) [:]

...

	... 228 more
Caused by: java.nio.file.FileSystemNotFoundException: Provider "vfs" not installed
	at java.nio.file.Paths.get(Paths.java:147) [rt.jar:1.8.0_152]
	at com.microsoft.playwright.impl.DriverJar.extractDriverToTempDir(DriverJar.java:90) [driver-bundle-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.DriverJar.initialize(DriverJar.java:44) [driver-bundle-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:52) [driver-1.19.0.jar:1.19.0]
	... 257 more

ERROR [default task-14] WFLYEJB0034: EJB Invocation failed on component PdfRenderService for method public byte[] au.com.infomedix.pdfkit.service.PdfRenderService.render(au.com.infomedix.pdfkit.dto.HtmlToPdfRequest) throws java.io.IOException: javax.ejb.EJBTransactionRolledbackException: Failed to create driver

...

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [rt.jar:1.8.0_152]
	at java.lang.Thread.run(Thread.java:748) [rt.jar:1.8.0_152]
Caused by: java.lang.RuntimeException: Failed to create driver
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:54) [driver-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.PlaywrightImpl.create(PlaywrightImpl.java:40) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:96) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:100) [playwright-1.19.0.jar:1.19.0]
	at au.com.infomedix.pdfkit.PlaywrightHtmlToPdf.render(PlaywrightHtmlToPdf.java:31) [:]

...
	at java.lang.Thread.run(Thread.java:748) [rt.jar:1.8.0_152]
Caused by: java.lang.RuntimeException: Failed to create driver
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:54) [driver-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.PlaywrightImpl.create(PlaywrightImpl.java:40) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:96) [playwright-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.Playwright.create(Playwright.java:100) [playwright-1.19.0.jar:1.19.0]
	at au.com.infomedix.pdfkit.PlaywrightHtmlToPdf.render(PlaywrightHtmlToPdf.java:31) [:]

...
	at org.jboss.as.ejb3.tx.CMTTxInterceptor.invokeInCallerTx(CMTTxInterceptor.java:255) [wildfly-ejb3-11.0.0.Final.jar:11.0.0.Final]
	... 228 more
Caused by: java.nio.file.FileSystemNotFoundException: Provider "vfs" not installed
	at java.nio.file.Paths.get(Paths.java:147) [rt.jar:1.8.0_152]
	at com.microsoft.playwright.impl.DriverJar.extractDriverToTempDir(DriverJar.java:90) [driver-bundle-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.DriverJar.initialize(DriverJar.java:44) [driver-bundle-1.19.0.jar:1.19.0]
	at com.microsoft.playwright.impl.Driver.ensureDriverInstalled(Driver.java:52) [driver-1.19.0.jar:1.19.0]
	... 257 more
```

I set a breakpoint and noticed that the exception is thrown from Java 1.8 `java.nio.Path.get()` method

```java
    public static Path get(URI uri) {
        String scheme =  uri.getScheme();
        if (scheme == null)
            throw new IllegalArgumentException("Missing scheme");

        // check for default provider to avoid loading of installed providers
        if (scheme.equalsIgnoreCase("file"))
            return FileSystems.getDefault().provider().getPath(uri);

        // try to find provider
        for (FileSystemProvider provider: FileSystemProvider.installedProviders()) {
            if (provider.getScheme().equalsIgnoreCase(scheme)) {
                return provider.getPath(uri);
            }
        }

        throw new FileSystemNotFoundException("Provider \"" + scheme + "\" not installed");
    }
```

The problematic `scheme` is  `vfs:/content/pdf-rest-server.war/WEB-INF/lib/driver-bundle-1.19.0.jar/driver/linux/`

The same code above ran successfully as a standalone Java application (i.e. not deployed inside an application server such as WildFly/JBoss).

--- a/driver-bundle/src/test/java/com/microsoft/playwright/TestInstall.java
+++ b/driver-bundle/src/test/java/com/microsoft/playwright/TestInstall.java
@@ -28,6 +28,9 @@ import java.nio.file.Path;
 import java.util.Collections;
 import java.util.concurrent.TimeUnit;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class TestInstall {
@@ -36,6 +39,8 @@ public class TestInstall {
     // Clear system property to ensure that the driver is loaded from jar.
     System.clearProperty("playwright.cli.dir");
     System.clearProperty("playwright.driver.tmpdir");
+    // Clear system property to ensure that the default driver is loaded.
+    System.clearProperty("playwright.driver.impl");
   }
 
   @Test
@@ -56,5 +61,20 @@ public class TestInstall {
     System.setProperty("playwright.driver.tmpdir", tmpdir.toString());
     DriverJar driver = new DriverJar();
     assertTrue(driver.driverPath().startsWith(tmpdir), "Driver path: " + driver.driverPath() + " tmp: " + tmpdir);
+  }
+
+  @Test
+  void playwrightDriverDefaultImpl() {
+    assertDoesNotThrow(() -> Driver.ensureDriverInstalled(Collections.emptyMap(), false));
+  }
+
+  @Test
+  void playwrightDriverAlternativeImpl() {
+    System.setProperty("playwright.driver.impl", "com.microsoft.playwright.impl.AlternativeDriver");
+    RuntimeException thrown =
+      assertThrows(
+        RuntimeException.class,
+        () -> Driver.ensureDriverInstalled(Collections.emptyMap(), false));
+    assertEquals("Failed to create driver", thrown.getMessage());
   }
 }

--- a/driver/src/main/java/com/microsoft/playwright/impl/Driver.java
+++ b/driver/src/main/java/com/microsoft/playwright/impl/Driver.java
@@ -98,7 +98,9 @@ public abstract class Driver {
       return new PreinstalledDriver(Paths.get(pathFromProperty));
     }
 
-    Class<?> jarDriver = Class.forName("com.microsoft.playwright.impl.DriverJar");
+    String driverImpl =
+      System.getProperty("playwright.driver.impl", "com.microsoft.playwright.impl.DriverJar");
+    Class<?> jarDriver = Class.forName(driverImpl);
     return (Driver) jarDriver.getDeclaredConstructor().newInstance();
   }
 

